spring-cloud-starter-stream-source-http/src/main/resources/config/application.yml

```diff
@@ -0,0 +1,5 @@
+spring:
+  cloud:
+    app-starter-stream:
```

Comment:
I think we want to change this to `stream-app-starters`, no?

Reply:
fixed